codex-rs/tui/src/app.rs

```diff
@@ -21,6 +22,22 @@ use std::path::PathBuf;
 use std::sync::mpsc::Receiver;
 use std::sync::mpsc::channel;
 
+/// Template for compact summary continuation prompt
+const COMPACT_SUMMARY_TEMPLATE: &str = concat!(
+    "This chat is a continuation of a previous conversation. ",
+    "After providing the summary, acknowledge that /compact command has been applied. ",
+    "Here is the summary of the previous conversation:\n\n{}"
```

Comment:
Using `{}` as a placeholder in this way seems very confusing to me as a Rust person because it's not being used natively by `format!()`. Please use something like `SUMMARY_TEXT` instead so it's more obvious that something is meant to be replaced.